Solutions/MISP2Sentinel/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-misp2sentinel",
+	"firstPublishDate": "2023-07-29",
+	"lastPublishDate": "2023-07-29",
+	"providers": ["MISP project", "cudeso.be"],

```

Comment:
As per the standards we can have only one "providers". Please update accordingly.

Reply:
changed in https://github.com/Azure/Azure-Sentinel/pull/8658/commits/01daea3dfc141fed17c97beae3d61d04e16749cb